code/game/machinery/hologram.dm

```diff
@@ -64,6 +70,15 @@ Possible to do for anyone motivated enough:
 
 	for (var/I in masters)
 		clear_holo(I)
+
+	if(replay_mode)
+		replay_stop()
+	if(record_mode)
+		record_stop()
+	
+	if(disk)

```

Comment:
this if is not really needed, also is this destroy double idented or is it just github

Reply:
Will fix, and just github.